package.json

```diff
@@ -25,12 +25,15 @@
     "djv": "^2.1.4",
     "jasmine": "^4.3.0",
     "moment": "~2.29.4",
-    "swagger-ui-dist": "^4.13.0",
+    "swagger-ui-dist": "^5.18.2",
     "vis": "^4.21.0"
   },
   "devDependencies": {
     "http-server": "^14.1.1"
   },
+  "scarfSettings": {
```

Comment:
What is this for?

Reply:
It's mentioned in the change description, but briefly it's to opt out of anonymised installation stats gathering.  See [readme for 5.18.2](https://yarnpkg.com/package?q=swagger-ui-dist&name=swagger-ui-dist&version=5.18.2).